index.html

```diff
@@ -16,13 +17,221 @@
 </head>
 
 <body>
+    <div class="container-fluid sticky-top col-12">
+    <div class="row">
+      <div class="col-sm-12 mt-4 d-flex justify-content-center">
+        <h1 class="logo-heading">SpaceWalkers</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-md-12 mt-4 d-flex justify-content-center">
+        <nav class="nav">
+          <a class="nav-link active" href="#">Home</a>
```

Comment:
I see that you added an active here but not to the others. But good thing to note is that when I clicked on the home button, it took me to it! 👍